firmware/baseband/proc_capture.hpp

```diff
@@ -55,6 +55,7 @@ class CaptureProcessor : public BasebandProcessor {
      * use decim_0_4 for an overall decimation factor of 8.
      * use decim_0_8 for an overall decimation factor of 16. */
     dsp::decimate::FIRC8xR16x24FS4Decim4 decim_0_4{};
```

Comment:
We should end up making a "composite decimator" type that takes all these decimators packs them into a std::variant -- once you have wrapped up your investigation, I'll work on refactoring to save space.

Reply:
I fully  agree,  but I will need your help on that ,